spec/requests/users_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) do
+      get '/users'
+    end
+    it 'returns http success for users#index action' do
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Here will be a list of users.')
+    end
+  end
```

Comment:
- [ ] Its a requirement that we make sure that for each action we check If a correct template was rendered. Kindly do so for the index action by adding a test case as we have done for the show action.